content/publishing/firebase-app-distribution.md

```diff
@@ -0,0 +1,35 @@
+---
+description: Deploy a Flutter app to Firebase App Distribution using the Flutter workflow editor
+title: Firebase App Distribution
+weight: 3
+---
+
+Set up publishing to [Firebase App Distribution](https://firebase.google.com/docs/app-distribution) to distribute your Android and iOS apps to testers.
+
+{{<notebox>}}
+This guide only applies to workflows configured with the **Flutter workflow editor**. If your workflow is configured with **codemagic.yaml**, please go to [Publishing an app to Firebase App Distribution](../publishing-yaml/distribution/#publishing-an-app-to-firebase-app-distribution).
+{{</notebox>}}
+
+## Requirements
+
+* [Add Firebase to your Flutter project](https://firebase.google.com/docs/flutter/setup?platform=ios)
+* Generate a [Firebase token](https://firebase.google.com/docs/cli#cli-ci-systems) locally
+* Set up [iOS code signing](../code-signing/ios-code-signing). An Ad Hoc or Enterprise distribution profile is required to distribute the app outside your development team.
```

Comment:
development profile works too (ad-hoc and enterprise is only required if you want to publish to people outside of your dev circle)

Reply:
maybe good to add this back in somewhere too (like on android)

{{<notebox>}} In addition, each uploaded binary must have a different version to appear in Firebase console, see how to automatically [increment build version](../building/build-versioning/ 'Build versioning') on Codemagic. {{</notebox>}}